app/src/ble.c

```diff
@@ -48,7 +50,7 @@ static uint8_t passkey_digit = 0;
 #endif /* IS_ENABLED(CONFIG_ZMK_BLE_PASSKEY_ENTRY) */
 
 #if IS_ENABLED(CONFIG_ZMK_SPLIT_ROLE_CENTRAL)
-#define PROFILE_COUNT (CONFIG_BT_MAX_PAIRED - 1)
+#define PROFILE_COUNT (CONFIG_BT_MAX_PAIRED - 1) // should it be ZMK_BLE_PROFILE_COUNT?
```

Comment:
They are functionally the same, one pairing slot is reserved for the right half

Reply:
Yes, but I can't find where `PROFILE_COUNT` was used anywhere in the code. Seems everywhere we are referring to `ZMK_BLE_PROFILE_COUNT` which made me a bit confused.

Reply:
The number of profiles configured in ZMK is not the same as the technical limitation on the number of profiles. The PROFILE_COUNT value is defined as runtime based on the max pairing slots configured, you can have 8 pairing slots and then use ZMK_BLE_PROFILE_COUNT to limit the number of actual profiles